test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -582,6 +582,9 @@ class test_uct_cm_sockaddr_multiple_cms : public test_uct_cm_sockaddr {
 
     void cleanup() {
         test_uct_cm_sockaddr::cleanup();
+
+        m_test_cm.reset();
```

Comment:
should be before calling  test_uct_cm_sockaddr::cleanup();

Reply:
why? 
test_uct_cm_sockaddr::cleanup() will destroy the ep on the server and then we can destroy the cm it was using

Reply:
order of destruction should be opposite of order of construction

Reply:
yes but in this case it's a new cm that is created in the middle of the flow..

Reply:
Ok so I'll restore server's ep destroy at the end of the test and reset the new cm before the base class cleanup